bftengine/src/bcstatetransfer/RVBManager.cpp

```diff
@@ -622,10 +622,10 @@ void RVBManager::computeDigestOfBlock(const uint64_t block_id,
                                       char* out_digest) const {
   ConcordAssertGT(block_id, 0);
   ConcordAssertGT(block_size, 0);
-  DigestUtil::Context c;
-  c.update(reinterpret_cast<const char*>(&block_id), sizeof(block_id));
-  c.update(block, block_size);
-  c.writeDigest(out_digest);
+  DigestGenerator digestGenerator;
```

Comment:
In this file the variable names are having underscores. So in this file the name should be digest_generator;

Reply:
Addressed in https://github.com/arc-vmware/concord-bft/pull/3/commits/b7fc8a50c49f6f986000ec33fefe4d8e1580802d